ci/push-image-step.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - task: Docker@1
+    displayName: Push image
+    inputs:
+      command: push
+      imageName: $(Build.Repository.Name):$(Build.BuildId)
```

Comment:
Can we use better tags than `1`, `2` ... `n`?

We're depending on the coincidence that the GitHub and Docker account share the same name. It could be confusing if that is ever not true.